src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export function Tabs({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  function hasSameTabId(tabId, selTabId) {
+    return tabId === selTabId;
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                key={tab.id}
+                className={cn({
+                  'is-active':
+                    hasSameTabId(id, selectedTab.id),
+                })
+                }
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (!hasSameTabId(id, selectedTabId)) {
+                      onTabSelected(tab);
+                    }
+                  }}
```

Comment:
it is bad practise to make anonymous arrow functions in jsx.
make handler in component and pass reference here